extension/js/common/composer.ts

```diff
@@ -148,6 +149,7 @@ export class Composer {
   private btnUpdateTimeout?: number;
   private refBodyHeight?: number;
   private urlParams: ComposerUrlParams;
+  private isMaximized = false;
```

Comment:
It would be good to group similar functionality together to be next to each other and named similarly.

For example you can put `isMaximized` next to `composeWindowIsMinimized` and rename `composeWindowIsMinimized` to `isMinimized`. Please do the same (renaming and grouping) with the other related functions if needed, so that the names turn out consistent for similar functionality.